wallet/src/main/scala/org/bitcoins/wallet/api/WalletApi.scala

```diff
@@ -50,6 +58,10 @@ sealed trait WalletApi {
   */
 trait LockedWalletApi extends WalletApi {
 
+  // TODO calculate one based off relevant data
```

Comment:
could you file an issue for this if there is not one already?

Reply:
#410